### PR TITLE
Add partial support of extensions for v17:

### PR DESCRIPTION
**Pull request by lubennikovaav**
- PostGIS 3.5.0
- pgrouting 3.6.2
- h3 4.1.3
- unit 7.9
- pgjwt version (f3d82fd)
- pg_hashids 1.2.1
- ip4r 2.4.2
- prefix 1.2.10
- postgresql-hll 2.18
- pg_roaringbitmap 0.5.4
- pg-semver 0.40.0

update support of extensions for v14-v16:
- unit 7.7 -> 7.9
- pgjwt 9742dab -> f3d82fd

### CODE REVIEW

File: compute/Dockerfile.compute-node [hunk 1]
```diff
@@ -109,32 +109,62 @@ RUN apt update && \
...
+    echo "${POSTGIS_CHECKSUM} postgis.tar.gz" | sha256sum --check && \
     mkdir postgis-src && cd postgis-src && tar xzf ../postgis.tar.gz --strip-components=1 -C . && \
     find /usr/local/pgsql -type f | sed 's|^/usr/local/pgsql/||' > /before.txt &&\
     ./autogen.sh && \
     ./configure --with-sfcgal=/usr/local/bin/sfcgal-config && \
+    make -j $(getconf _NPROCESSORS_ONLN) && \
     make -j $(getconf _NPROCESSORS_ONLN) install && \
     cd extensions/postgis && \
     make clean && \
+    make -j $(getconf _NPROCESSORS_ONLN) && \
```
Comment (hlinnaka) [inline compute/Dockerfile.compute-node:167]: Why are these needed? Does `make install` not build first?

File: compute/Dockerfile.compute-node [hunk 2]
```diff
@@ -445,10 +501,9 @@ FROM build-deps AS hll-pg-build
 ARG PG_VERSION
 COPY --from=pg-build /usr/local/pgsql/ /usr/local/pgsql/
 
-RUN case "${PG_VERSION}" in "v17") \
-    echo "v17 extensions are not supported yet. Quit" && exit 0;; \
-    esac && \
-    wget https://github.com/citusdata/postgresql-hll/archive/refs/tags/v2.18.tar.gz -O hll.tar.gz && \
+# not version-specific
+# last release v2.18 - Sep 15, 2023
```
Comment (hlinnaka) [inline compute/Dockerfile.compute-node:505]:
```suggestion
# last release v2.18 - Aug 29, 2023
```
according to https://github.com/citusdata/postgresql-hll/blob/master/CHANGELOG.md. According to github, the tag was created Aug 31, 2023.

File: compute/Dockerfile.compute-node [hunk 3]
```diff
@@ -109,32 +109,62 @@ RUN apt update && \
...
-    mkdir -p /sfcgal && \
-    echo "Postgis doensn't yet support PG17 (needs 3.4.3, if not higher)" && exit 0;; \
+# and also we must check backward compatibility with older versions of PostGIS.
+#
+# Use new version only for v17
+RUN case "${PG_VERSION}" in \
+    "v17") \
+        export SFCGAL_VERSION=1.4.1 \
+        export SFCGAL_CHECKSUM=1800c8a26241588f11cddcf433049e9b9aea902e923414d2ecef33a3295626c3 \
+    ;; \
```
[the remark this answers is not in the document]
Comment (lubennikovaav) [reply compute/Dockerfile.compute-node:124]: I've tried to build sfcgal 1.5, but it depends on CGAL version that is not present in Bookworm and I didn't manage to build them both from source.
The build was successful, but I got strange error on `create extension postgis_sfcgal` :
```
could not load library "/usr/local/lib/postgis_sfcgal-3.so": /usr/local/lib/postgis_sfcgal-3.so: undefined symbol: sfcgal_geometry_visibility_point.
```
using sgcgal 1.4 fixed this.
I think for now this is good enough to ship postgis for v17.

File: compute/Dockerfile.compute-node [hunk 4]
```diff
@@ -215,10 +261,27 @@ FROM build-deps AS h3-pg-build
...
+      *) \
+        echo "Unsupported architecture '$(uname -m)'. Supported are x86_64 and aarch64" && exit 1 \
+        ;; \
     esac && \
+    wget https://github.com/Kitware/CMake/releases/download/v3.24.2/cmake-3.24.2-linux-$(uname -m).sh \
+      -q -O /tmp/cmake-install.sh \
+      && echo "${CMAKE_CHECKSUM} /tmp/cmake-install.sh" | sha256sum --check \
+      && chmod u+x /tmp/cmake-install.sh \
+      && /tmp/cmake-install.sh --skip-license --prefix=/usr/local/ \
+      && rm /tmp/cmake-install.sh
```
[the remark this answers is not in the document]
Comment (lubennikovaav) [reply compute/Dockerfile.compute-node:280]: thank you! missed this while resolving merge conflicts

File: compute/Dockerfile.compute-node (hunk 1 above)
Comment (lubennikovaav) [reply compute/Dockerfile.compute-node]: 1. without separate `make` it failed with this error:
```
44.15 make[1]: Entering directory '/postgis-src/loader'
44.15 /bin/bash ../libtool --mode=install /usr/bin/install -c pgsql2shp "/usr/local/bin/pgsql2shp"
44.19 libtool: install: /usr/bin/install -c pgsql2shp /usr/local/bin/pgsql2shp
44.19 /usr/bin/install: cannot stat 'pgsql2shp': No such file or directory
```

we don't really use this submodule
```
RUN cd /usr/local/pgsql/bin && rm -f ecpg raster2pgsql shp2pgsql pgtopo_export pgtopo_import pgsql2shp
```
but it blocks the build. so I added make step.

2. removed second make, this is indeed not needed

**Review by hlinnaka — APPROVED** [review]
